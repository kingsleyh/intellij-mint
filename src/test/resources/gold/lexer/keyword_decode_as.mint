module A {

  fun decoder() : String {
    decode as String
  }

}