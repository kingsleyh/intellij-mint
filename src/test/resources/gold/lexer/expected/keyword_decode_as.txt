module ('module')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('decoder')
( ('(')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('String')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
decode ('decode')
WHITE_SPACE (' ')
as ('as')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('String')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')